Calculator.Tests/Calculator.Tests/CalculatorTests.cs

```diff
@@ -0,0 +1,248 @@
+﻿using System;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Support.UI;
+
+namespace Calculator.Tests
+{
+    public class CalculatorTests
+    {
+        IWebDriver browser;
+
+        [SetUp]
+        public void BeforeEachTest()
+        {
+            browser = new ChromeDriver();
+            browser.Url = "http://127.0.0.1:8080";
+            browser.Manage().Timeouts().PageLoad = TimeSpan.FromSeconds(60);
+            browser.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(10);
+            browser.FindElement(By.Id("login")).SendKeys("test");
```

Comment:
I suggest adding an empty line here.

```suggestion
            browser.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(10);
            
            browser.FindElement(By.Id("login")).SendKeys("test");
```

Reply:
done